app/controllers/api/reservations_controller.rb

```diff
@@ -4,5 +4,62 @@ def index
       @reservations = current_user.reservations.all
```

Comment:
[SUGGESTION]: Please consider replacing

```ruby
@reservations = current_user.reservations.all
```

with

```ruby
@reservations = current_user.reservations
```

as it returns the same array of all the reservations. Appending the `all` operation doesn't change a thing.